From: Marcus Barrow <mbarrow@redhat.com>
Date: Wed, 1 Oct 2008 12:24:11 -0400
Subject: [scsi] qla2xxx: new version string defintion
Message-id: 20081001162411.18756.52834.sendpatchset@file.bos.redhat.com
O-Subject: [rhel 5.3 patch] qla2xxx - Update version to agree with open source standards.
Bugzilla: 465023

BZ 465023

The old string was: 8.02.00-k5-rhel5.3-02, the new more parseable one
is: 8.02.00.03.05.03

qla2xxx: Update version to agree with open source standards.

diff --git a/drivers/scsi/qla2xxx/qla_version.h b/drivers/scsi/qla2xxx/qla_version.h
index c57c115..710e3b4 100644
--- a/drivers/scsi/qla2xxx/qla_version.h
+++ b/drivers/scsi/qla2xxx/qla_version.h
@@ -7,7 +7,7 @@
 /*
  * Driver version
  */
-#define QLA2XXX_VERSION      "8.02.00-k5-rhel5.3-01"
+#define QLA2XXX_VERSION      "8.02.00.03.05.03"
 
 #define QLA_DRIVER_MAJOR_VER	8
 #define QLA_DRIVER_MINOR_VER	1
